tests/cli/plugin/plugin_create_tests.py

```diff
@@ -0,0 +1,52 @@
+import os
+
+from core.base_test.tns_test import TnsTest
+from core.enums.os_type import OSType
+from core.settings import Settings
+from core.utils.file_utils import Folder
+from products.nativescript.tns import Tns
+
+
+class PluginCreateTests(TnsTest):
+    plugin_name = "nativescript-test-plugin"
+
+    @classmethod
+    def setUpClass(cls):
+        TnsTest.setUpClass()
+
+    def setUp(self):
+        TnsTest.setUp(self)
+
+    def tearDown(self):
+        TnsTest.tearDown(self)
+        Folder.clean(self.plugin_name)
+        Folder.clean("plugin-folder")
+
+    @classmethod
+    def tearDownClass(cls):
+        TnsTest.tearDownClass()
+        Folder.clean(cls.plugin_name)
+
+    def test_100_plugin_create(self):
+        Tns.plugin_create(plugin_name=self.plugin_name)
+
+    def test_200_plugin_create_with_path(self):
+        Tns.plugin_create(plugin_name=self.plugin_name, path="plugin-folder", verify=False)
+
+    def test_201_plugin_create_custom_template(self):
+        result = Tns.exec_command(command="plugin create --includeTypeScriptDemo=y nativescript-test-plugin "
```

Comment:
It we run it like this we lose all the checks in verify of `plugin_create` method.
Can we somehow address it.

May be adding `template` as param of `plugin_create`?